### ISSUE TITLE
All results are being rejected

### ISSUE THREAD

**Issue by guiprz** [fireice-uk/xmr-stak#1644]
Basically this. I'm trying to mine but I only get rejected results. I extracted the newest version in a new file btw, there is no interference with the old version.

# Basic information
  - i5 6300HQ
  - GTX 960M 4GB.
 - Windows 10

**Comment by borzaka** [comment]
Give us more info please;
- What coin are you mining?
- What algorithm have you selected for it?
- What pool are you using, with what settings?

Probably you have selected a wrong algo for the coin you are mining.

**Comment by guiprz** [comment]
Oh yes, sorry. I'm mining Electroneum. But shouldn't the miner select the CN7 algorithm automatically? I'm using easyhash pool on a low difficulty port, tested with other pools and they didn't work too.

**Comment by borzaka** [comment]
No, it's not auto, you have to specify which algorithm you want to use. When you run XMR-Stak for the first time, you are asked some questions.

For Electroneum you have to select `cryptonight_v7`, and make sure that the used pool is upgraded to the latest node version.

**Comment by guiprz** [comment]
Oh so now when it asks about what currency I want to mine I simply write cryptonight_v7 instead of electroneum? Gonna try it.

**Comment by daletho** [comment]
I just had the same problem after upgrading from 4.4.2 to 4.4.4. It ran fine for +- 12 hrs then only returning "Result rejected by the pool" for the next 17 hrs. Mining against supportxmr with currency set to monero7. Using my own build.

One other thing I noticed which may or may not be related is my backup pool monerohash wont even log in with this build of 4.4.4, it just gives "SOCKET ERROR - [monerohash.com:9999] RECEIVE error: socket closed". However my 4.4.2 build still works find with the exact same config files.

I don't have much time at the moment to troubleshoot so I'm back on 4.2.2 as it has worked well for me since its initial release, just adding this info if it can help.

**Comment by psychocrypt** [comment]
There is not version 4.4.4 available. Please post the output of `xmr-stak
-V`

**Comment by daletho** [comment]
Sorry my mistake! meant 2.*:
Version: xmr-stak/2.4.4/c0ab1734/unknown/win/amd-cpu/aeon-cryptonight-monero/0

**Comment by daletho** [comment]
A snippet of the log below, There was a timeout with supportxmr followed by 17 hrs worth of "Result rejected by the pool.", restarting the client resulted in the pool accepting results again.

[2018-06-08 19:40:10] : Result accepted by the pool.
[2018-06-08 19:40:11] : Result accepted by the pool.
[2018-06-08 19:41:57] : Difficulty changed. Now: 171750.
[2018-06-08 19:41:57] : New block detected.
[2018-06-08 19:42:02] : Result accepted by the pool.
[2018-06-08 19:42:09] : Result accepted by the pool.
[2018-06-08 19:43:35] : New block detected.
[2018-06-08 19:43:39] : New block detected.
[2018-06-08 19:43:42] : Result accepted by the pool.
[2018-06-08 19:43:51] : Result accepted by the pool.
[2018-06-08 19:44:22] : Fast-connecting to monerohash.com:9999 pool ...
[2018-06-08 19:44:22] : SOCKET ERROR - [pool.supportxmr.com:7777] CALL error: Timeout while waiting for a reply
[2018-06-08 19:45:17] : SOCKET ERROR - [monerohash.com:9999] Unknown TLS error. Secure TLS maybe unsupported, try setting tls_secure_algo to false.
[2018-06-08 19:45:17] : Fast-connecting to pool.supportxmr.com:7777 pool ...
[2018-06-08 19:45:17] : Pool pool.supportxmr.com:7777 connected. Logging in...
[2018-06-08 19:45:51] : Result rejected by the pool.
[2018-06-08 19:46:12] : Result rejected by the pool.
[2018-06-08 19:46:36] : Result rejected by the pool.
[2018-06-08 19:47:40] : Result rejected by the pool.

**Comment by KnoAll** [comment]
I'm seeing this as well. i am on 2.4.5 using xmr-us-west1.nanopool.org:14433 and xmr-us-east1.nanopool.org:14433 with the same results
